src/docfx/build/toc/parser/MarkdownTocMarkup.cs

```diff
@@ -102,6 +102,12 @@ TableOfContentsInputItem GetItem(HeadingBlock block)
                     return currentItem;
                 }
 
+                if (block.Inline.Count() > 1 && block.Inline.Any(l => l is XrefInline || l is LinkInline))
+                {
+                    errors.Add(Errors.InvalidTocSyntax(new Range(block.Line, block.Column), filePath, tocContent.Substring(block.Span.Start, block.Span.Length), "multiple inlines in one heading block is not allowed"));
```

Comment:
It may be hard for writers to understand terms like `inline` and `heading block`.

Reply:
it's a hint, but the last message is important, referencing to the docs.